File: C# Parctices/Leap Year.cs
using System;  
namespace CheckLeapYear  
{  
    class Program   
    {  
        static void Main(string[] args)  
        {

            int year;
	        year = Convert.ToInt32(Console.ReadLine());

            if(year%400==0)
                Console.WriteLine("Leap Year!");

            else if(year%4==0 && year%100!=0)
                Console.WriteLine("Leap Year!");

            else
                Console.WriteLine("Not Leap Year!");
        }
    }
}

File: basic-cpp-practice/Leap year.cpp
#include<bits/stdc++.h>
using namespace std;
int main()
{
    int T, year;
    cin >> T;

    while(T--)
    {
        cin >> year;

        if(year%400==0)
            cout << "Leap Year!" << endl;

        else if(year%4==0 && year%100!=0)
            cout << "Leap Year!" << endl;

        else
            cout << "Not Leap Year!" << endl;
    }
    return 0;

}
